README.md

```diff
@@ -2,7 +2,7 @@
 
 ```sh
 brew tap wrf-cmake/wrf
-brew install wrf -v
+brew install wrf --HEAD -v
```

Comment:
It's your tap, so take my advice or leave it, my feelings won't be hurt 😄 

I'd recommend adding an explanation here as to why they should pass `--HEAD`. (As described in #3)

Reply:
Thanks -- sure, will make a note for now until we release 4.1.0